tests/testthat/test-osmodel.R

```diff
@@ -0,0 +1,32 @@
+
+test_that("parametrize filled all the gaps", {
+    os_mod <- LogLogisticOs()
+    h_link <- HazardLink(
+        stan = StanModule(
+            functions = "bla",
+            data = "bla",
+            parameters = "bla",
+            transformed_parameters = "bla",
+            generated_quantities = "bla",
+            priors = list(),
+            inits = list()
+        ),
+        parameters = "bla",
+        contribution = "bla"
+    )
+    os_mod_f <- parametrize(os_mod, h_link)
+
+    expect_true(all(sapply(
+        c("link_arguments", "link_log_hazard_contribution", "link_arguments_as_par"),
+        grepl, os_mod_f@stan@functions
+    )) != TRUE)
```

Comment:
Instead of `!= TRUE` could you not just use `expect_false()` ?

Reply:
Yes. My mistake. Fixed